python/tests/test_broadcast.py

```diff
@@ -0,0 +1,44 @@
+# Copyright © 2025 Apple Inc.
+
+import mlx.core
+import mlx_tests
+
+
+class TestBroadcast(mlx_tests.MLXTestCase):
+    def test_broadcast_shapes(self):
```

Comment:
I don't think we need a separate file for this test since I don't see us adding anything here. Just put it in the ops test for now.

Reply:
Right. I moved `broadcast_shapes` test into `test_ops.py`